e2e/interchaintestv8/relayer/eth_and_cosmos_config.go

```diff
@@ -25,6 +25,8 @@ type EthCosmosConfigInfo struct {
 	SignerAddress string
 	// Whether we use the mock client in Cosmos
 	MockWasmClient bool
+	// Log level for the relayer
+	LogLevel string
```

Comment:
Log level is only for human readability and has no impact on if or how the e2es pass. So it seems a bit weird for it to be parameterized this way. But I'm fine with it.

Reply:
Sure, but it was hard to set when running the tests before, so having the ability to modify it without changing code every time seems reasonable